include/mapnik/json/unicode_string_grammar_x3.hpp

```diff
@@ -34,14 +34,12 @@ namespace mapnik { namespace json { namespace grammar {
 namespace x3 = boost::spirit::x3;
 using unicode_string_grammar_type = x3::rule<class unicode_string_tag, std::string>;
 using escaped_unicode_type = x3::rule<class escaped_unicode_tag, std::string>;
-
 BOOST_SPIRIT_DECLARE(unicode_string_grammar_type);
 BOOST_SPIRIT_DECLARE(escaped_unicode_type);
-
 }
 
 grammar::unicode_string_grammar_type const& unicode_string_grammar();
-grammar::escaped_unicode_type const& escaped_unicode_grammar();
+__attribute__((visibility("default"))) grammar::escaped_unicode_type const& escaped_unicode_grammar();
```

Comment:
setting visibility to "default" to make `escaped_unicode_grammar()` visible during linking stage.  I'm not sure if we need to expose more to make `libmapnik-json` more usable, lets see.

/cc @springmeyer

Reply:
I'm seeing that merge went wrong and some changes got overwritten, hmm.. on the case
/cc @springmeyer

Reply:
I'm having second thoughts on re-arranging grammars. The issue with duplicate symbols only manifest itself when linking plug-ins statically. Adding `escaped_unicode_grammar()` helps but as a result requires linking to `mapnik-json.a` in rundemo and `mapnik-render`. I might revert this change all together. 
/cc @springmeyer